trip_app/trips/tests/conftest.py

```diff
@@ -17,10 +19,12 @@ def trips(request):
 def trip_data():
     """ Create a trip model without touching database and return its dict
         repr for using as a request payload """
-    trip = TripFactory.build()
-    serializer = TripSerializer(trip)
-    data = serializer.data
-    del data["id"]
-    del data["driver"]
-    del data["passengers"]
+    data = {
+        "dep_time": (timezone.now() + timedelta(days=1)),
+        "start_point": f"{faker.latitude()} {float(faker.longitude())}",
```

Comment:
Could you explain why longitude cast to float and latitude not?
I checked, you can use `faker.latlng` function instead generating this individually.

Reply:
It is just copypast from me, sorry. Actually there is no need to cast to float in this case as I need str here and f-string would call the str method of each decimal with proper formatting, therefore I will remove cast at all.
Regarding `faker.latlng` it returns tuple of Decimal objects, which, I'm now sure how to convert to proper string that I need (though if you know how to do it please guide me with more details).